packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html

```diff
@@ -96,6 +96,7 @@
     <a
         class="oui-link_icon mt-3"
         data-ng-href="{{:: $ctrl.dashboardRedirectURL }}"
+        data-ng-click="$ctrl.gotToDashboard()"
```

Comment:
You could use the `data-track-*` directives
https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/general-information/general-information.html#L49-L57

Reply:
@y4nnL : could you explain a bit more what's the goal of `data-track-*` directives please

Reply:
if you have only to handle tracking data so consider using track directive.
if you have to complete others task during the event consider to use click event only to avoid multiple event binding.

Reply:
Done Thanks